.github/workflows/release_windows_packages.yml

```diff
@@ -71,143 +71,47 @@ jobs:
-          restore-keys: |
-            portable-linux-package-matrix-v1-${{ matrix.target_bundle.amdgpu_family }}-
+          python-version: '3.12'
 
-      - name: Install the AWS tool
-        run: ./dockerfiles/cpubuilder/install_awscli.sh
-
-      - name: Fetch sources
+      - name: Print status
         run: |
-          # Prefetch docker container in background.
-          docker pull ${{ env.BUILD_IMAGE }} &
-          ./build_tools/fetch_sources.py --jobs 10
-          wait
+          echo "DIST_ARCHIVE: ${DIST_ARCHIVE}"
+          echo "RELEASE_TYPE: ${RELEASE_TYPE}"
+          echo "S3_TARBALL: ${S3_TARBALL}"
 
-      - name: Build Projects
-        run: |
-          ./build_tools/linux_portable_build.py \
-            --image=${{ env.BUILD_IMAGE }} \
-            --output-dir=${{ env.OUTPUT_DIR }} \
-            -- \
-            "-DTHEROCK_AMDGPU_FAMILIES=${{ matrix.target_bundle.amdgpu_family }}"
-          cd ${{ env.OUTPUT_DIR }}/build/dist/rocm
-          echo "Building ${{ env.DIST_ARCHIVE }}"
-          tar cfz "${{ env.DIST_ARCHIVE }}" .
-
-      - name: Build Python Packages
-        run: |
-          ./build_tools/linux_portable_build.py \
-            --image=${{ env.BUILD_IMAGE }} \
-            --output-dir=${{ env.OUTPUT_DIR }}/packages \
-            --build-python-only \
-            --artifact-dir=${{ env.OUTPUT_DIR }}/build/artifacts \
-            -- \
-            "--version=${{ needs.setup_metadata.outputs.version }}"
-
-      - name: Upload Tarball to GitHub
-        uses: ncipollo/release-action@440c8c1cb0ed28b9f43e4d1d670870f059653174 # v1.16.0
-        with:
-          artifacts: "${{ env.DIST_ARCHIVE }}"
-          tag: "${{ env.RELEASE_TYPE }}-tarball"
-          name: "${{ env.RELEASE_TYPE }}-tarball"
-          body: "Automatic ${{ env.RELEASE_TYPE }} tarball of TheRock."
-          removeArtifacts: false
-          allowUpdates: true
-          replacesArtifacts: true
-          makeLatest: false
-
-      - name: Configure AWS Credentials
-        if: ${{ github.repository_owner == 'ROCm' }}
-        uses: aws-actions/configure-aws-credentials@ececac1a45f3b08a01d2dd070d28d111c5fe6722 # v4.1.0
-        with:
-          aws-region: us-east-2
-          role-to-assume: arn:aws:iam::692859939525:role/therock-${{ env.RELEASE_TYPE }}-releases
-
-      - name: Upload Releases to S3
-        if: ${{ github.repository_owner == 'ROCm' }}
-        run: |
-          aws s3 cp ${{ env.DIST_ARCHIVE }} s3://${{ env.S3_TARBALL }}
-          aws s3 cp ${{ env.OUTPUT_DIR }}/packages/dist/ s3://${{ env.S3_PYTHON }}/${{ matrix.target_bundle.amdgpu_family }}/ \
-          --recursive --no-follow-symlinks \
-          --exclude "*" \
-          --include "*.whl" \
-          --include "*.tar.gz"
-
-      - name: Setup Python
-        if: ${{ github.repository_owner == 'ROCm' }}
-        uses: actions/setup-python@8d9ed9ac5c53483de85588cdf95a591a75ab9f55 # v5.5.0
-        with:
-          python-version: 3.12
```

Comment:
Linux workflow could use some cleanup too, `python-version: '3.10'` is used above already.